### DISCUSSION TITLE
How do you know if the app is running correctly?

### DISCUSSION THREAD

**Discussion by natecodes** [jdholtz/auto-southwest-check-in#251] [General]
<img width="444" alt="image" src="https://github.com/jdholtz/auto-southwest-check-in/assets/37425659/9f5844cf-1bd5-41f3-994b-f7449998dfb2">

Essentially I'm wondering what the proper workflow is; is this running properly after hanging? And are we supposed to just minimize the terminal and never close/sleep our computer?

Can we run this as a background process `bg`?

**Comment by natecodes** [comment]
Also, let's say that you run the script and then put your laptop to sleep. Will the script notify you that timing is off-sync?

**Comment by jdholtz** [comment]
> Essentially I'm wondering what the proper workflow is; is this running properly after hanging? And are we supposed to just minimize the terminal and never close/sleep our computer?
>
> Can we run this as a background process bg?

Yes, from your screenshot it appears to be running correctly. It also works if you run this as a background process. As the script runs as long as you have flights (for a reservation) or forever (provided an account), you shouldn't sleep your computer as that will mess up the timing of the script's execution. This script performs best on a server that is always running. If that is not available to you, you can run the script a few hours in advance and still get the benefit of not needing to worry about checking into your flight at the exact time.

**Reply by jdholtz** [reply]
Hey @natecodes, does my reply above answer your question?

**Reply by natecodes** [reply]
It does in confirming that the app works correctly, but I can 100% confirm that I never slept the computer since I started the script. I'm wondering how it was off sync by 3 minutes.

Edit: I just remembered that I opened a separate issue for the off sync thing. This discussion can be closed since it is confirmed how to run the script.
